### DISCUSSION TITLE
La Decadanse (Geneva) is down

### DISCUSSION THREAD

**Discussion by MarkJaroski** [jakopako/croncert-config#406] [General]
I've avoided adding Geneva venues, because Geneva has (had) the wonderful La Decadanse website, which unfortunately is down, and doesn't seem to be coming back, at least for a few weeks.

So I think I'm going to add some Geneva venues now, and try as best I can to recreate it.

**Comment by MarkJaroski** [comment]
They're back. It looks like an upstream hosting problem.

**Reply by jakopako** [reply]
Glad to hear they're back :)
